tracing-subscriber/src/fmt/mod.rs

```diff
@@ -615,6 +615,19 @@ where
         }
     }
 
+     /// Whether to write errors to the Writer, or ignore them.
+     ///
+     /// An error in this case refers to errors which may occur within the
+     /// Subscriber implementation itself, or in code that it depends on.
+     ///
+     /// It does not refer to traces set to the level of Error.
```

Comment:
/// Sets whether to write errors from [`FormatEvent`] to the writer. Defaults to true.
    ///
    /// By default, `fmt::Subscriber` will write any `FormatEvent`-internal errors. These errors
    /// are unlikely and will only occur if there is a bug in the `FormatEvent` implementation.
    /// 
    /// [`FormatEvent`]: crate::fmt::FormatEvent

Reply:
Updated it now